src/pure.ts

```diff
@@ -23,6 +23,20 @@ export const loadStripe: LoadStripe & {setLoadParameters: SetLoadParams} = (
 };
 
 loadStripe.setLoadParameters = (params): void => {
+  // we won't throw an error if setLoadParameters is called with the same values as before
+  if (loadStripeCalled && loadParams) {
+    const parameters = Object.keys(params) as Array<keyof typeof params>;
```

Comment:
suggestion: can we validate the new params in the same way that we validate the previous call?

This will result in a better error message if `setLoadParameters` is called after script initialization with invalid params.

```suggestion
    const parameters = Object.keys(validateLoadParams(params));
```